rca/static_src/sass/components/_hero.scss

```diff
@@ -3,14 +3,17 @@
     overflow: hidden;
 
     &--medium {
-        max-height: 340px;
+        margin-top: 120px; // height of header
```

Comment:
Seems like the only way I could solve this. The only side effect is that users will see a white bar when they scroll up when they're at the top most part of the page (which I don't think people do):

Production:

![Screenshot 2025-04-24 at 1 41 33 PM](https://github.com/user-attachments/assets/eed0e8ee-0f41-4f76-91b0-317f310bf318)

This change:

![Screenshot 2025-04-24 at 1 41 36 PM](https://github.com/user-attachments/assets/2677b3d7-a9d6-44e2-bf80-9d61d7500382)

Reply:
As discussed on standup, we'll want to make the white bar to be black instead.